server/api/db/sqldb/models/models_mysql.py

```diff
@@ -172,6 +172,41 @@ class Notification(Base, mlrun.utils.db.BaseModel):
     return Notification
 
 
+def make_alert_state(table):
+    class AlertState(Base, mlrun.utils.db.BaseModel):
+        __tablename__ = f"{table}_alert_states"
```

Comment:
Why is this a generic table? It's creating only one table called `alert_configs_alert_states`. Are there going to be `alert_states` of an object that isn't an `alert_config`?
Feels a bit redundant to me